src/main/java/edu/harvard/iq/dataverse/workflows/review/Comment.java

```diff
@@ -0,0 +1,56 @@
+package edu.harvard.iq.dataverse.workflows.review;
+
+import edu.harvard.iq.dataverse.DatasetVersion;
+import java.io.Serializable;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+
+@Entity
+public class Comment implements Serializable {
```

Comment:
Here you go: https://github.com/IQSS/dataverse/blob/e57acda007f5be011edf2740ceea3e6e47ab6138/src/main/java/edu/harvard/iq/dataverse/workflows/review/PublicationAuditEntry.java#L44